lib/parliament/response.rb

```diff
@@ -0,0 +1,27 @@
+require 'forwardable'
+
+module Parliament
+  class Response
+    include Enumerable
+    extend Forwardable
+    attr_reader :nodes
+    def_delegators :@nodes, :size, :each, :select, :map, :select!, :map!, :count, :length
+
+    def initialize(nodes)
+      @nodes = nodes
+    end
+
+    def filter(*types)
+      filtered_objects = Array.new(types.size) { [] }
```

Comment:
If an empty array is passed to filter, or somehow filter objects equals `[]` should we still loop over all our nodes?